docs/wegue-configuration.md

```diff
@@ -19,6 +19,7 @@ This describes the Wegue application configuration, which is modelled as JSON do
 | **mapCenter**      | Initial center of the map in map projection | `"mapCenter": [0, 0]` |
 | mapProjection      | Configuration object for CRS / projection used for the map | see [mapProjection](wegue-configuration?id=mapprojection) |
 | mapGeodataDragDop  | Configuration object for geodata file drag/drop functionality on the map. Only by setting the config this function will be enabled. | see [mapGeodataDragDop](wegue-configuration?id=mapGeodataDragDop) |
+| mapHover           | Configuration object to customize the bevahior of hover tooltips on the map. | see [mapHover](wegue-configuration?id=mapHover)
```

Comment:
We can say this is simple detail level, but to keep the description in the same style as the other properties of the general section, wouldn't it be better to write it as `Configuration object for hover tooltips behavior on the map` or, like the `legend` one, `Configuration object containing application wide parameters for hover tooltips` ?

Reply:
This also had a bad typo. I chose your 2nd formulation :)

Reply:
Thanks for thinking about it !